lib/org/chromium/webidl/LocalGitRunner.js

```diff
@@ -119,6 +121,16 @@ foam.CLASS({
       value: true,
     },
     {
+      class: 'FObjectProperty',
+      of: 'foam.dao.DAO',
+      documentation: `If provided, all IDL outputs will be forwarded to given
+          DAO (for persistent storage).`,
+      name: 'outputDao',
+      factory: function() {
+        return this.dao;
```

Comment:
I do not know JS very well. Is there a reason that this just returns dao, but the one below returns dao or null?